content/docs/intro/concepts/how-pulumi-works.md

```diff
@@ -2,12 +2,12 @@
 title: "How Pulumi Works"
 expanded_url: /docs/reference/concepts/
```

Comment:
Right now, when I click "How Pulumi Works" in the TOC, the parent "Architecture & Concepts" item becomes collapsed. The reason is this line. It should be deleted and then the "Architecture & Concepts" item will remain expanded when viewing this page.

```suggestion
```

Applies to the other pages under "Architecture & Concepts". We should do a review of all `expanded_url`s to verify they're needed anymore.